sdk/tests/mt-c-tests/uart/uart.c

```diff
@@ -9,14 +9,15 @@
 #include <stdint.h>
 #include <errno.h>
 #include <string.h>
-#include <flexpret.h>
+#include <flexpret/flexpret.h>
 
-#define CLOCKFREQ     ((int)(CLOCK_FREQ_MHZ)) // 100 MHz
+#define CLOCKFREQ     ((int)(50e6)) // 100 MHz
```

Comment:
Here the comment and the valye doesnt match. Also, I don't like that its a floating point value casted to an int. Just do 50000000?

Reply:
That's weird, looks like I messed up a little with git. Will fix